### ISSUE TITLE
Blastomyces dermatitidis on sputum fungal stain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10990]
**Preferred term label:**
Blastomyces dermatitidis on sputum fungal stain

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Blastomycosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001220
- label: Positive Blastomyces dermatitidis sputum fungal stain
- parents: [Sputum infectious agent (HP:0430107)](https://hpo.jax.org/app/browse/term/HP:0430107)
- pmid: PMID:28723016
- def: Demonstration of Blastomyces dermatitidis by microscopic examination of a stained sputum specimen.
- comment: Blastomycosis is a fungal infection caused by Blastomyces dermatitidis. It presents as a pulmonary infection after the inhalation of spores, and it may be either asymptomatic or have severe life-threatening complications like acute respiratory distress syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10990
- synonyms: Blastomyces dermatitidis on sputum fungal stain(NONE)
